sci-mathematics/calc/calc-2.12.6.7.recipe

```diff
@@ -43,8 +46,8 @@ fi
 
 PROVIDES_devel="
 	calc${secondaryArchSuffix}_devel = $portVersion
-	devel:libcalc$secondaryArchSuffix
-	devel:libcustcalc$secondaryArchSuffix
+	devel:libcalc$secondaryArchSuffix = $libVersionCompat
+	devel:libcustcalc$secondaryArchSuffix = $libVersionCompat
```

Comment:
```diff
-	devel:libcalc$secondaryArchSuffix = $libVersionCompat
-	devel:libcustcalc$secondaryArchSuffix = $libVersionCompat
+	devel:libcalc$secondaryArchSuffix = $libVersion
+	devel:libcustcalc$secondaryArchSuffix = $libVersion
```

Later, we should fix `BUILD_REQUIRES`:
```diff
 	devel:libncurses$secondaryArchSuffix
 	"
+if [ -n "$secondaryArchSuffix" ]; then
+BUILD_REQUIRES+="
+	calc_data == $portVersion
+	"
+fi
 BUILD_PREREQUIRES="
```
This prevents a build for a secondary arch from failing if an old release of `calc_data` is still installed.
Of course, if we build calc, then install it, and then build calc_x86, the calc_data package gets updated.
But if we choose to build calc and calc_x86 before installing calc, then we need the above hack to avoid a build error.